crest/Assets/Crest/Crest/Scripts/LodData/Settings/SimSettingsAnimatedWaves.cs

```diff
@@ -23,6 +23,12 @@ public partial class SimSettingsAnimatedWaves : SimSettingsBase
 
         public const string k_HelpURL = Internal.Constants.HELP_URL_BASE_USER + "ocean-simulation.html" + Internal.Constants.HELP_URL_RP + "#animated-waves-settings";
 
+        [Tooltip("PREVIEW: Set this to 2 to improve wave quality. In some cases like flowing rivers this can make a substantial difference to visual stability."
+            + "\n\nWe recommend doubling the Lod Data Resolution on the OceanRenderer component to preserve detail after making this change, but note that this will "
+            + "consume 4x more video memory until we are able to optimise data usage further, so apply this change with caution."), SerializeField]
+        float _waveResolutionMultiplier = 1f;
```

Comment:
Should this be a range and have protection against divide-by-zero?

Reply:
Yeah good shout done now